### DISCUSSION TITLE
DNS server not authoritative for zone.

### DISCUSSION THREAD

**Discussion by nqm322** [aircrack-ng/aircrack-ng#2590] [Q&A]
I getting error : DNS server not authoritative for zone, i tried to fix it different ways but still not working. I getting this error when i try to use comands airmon -ng, ls, and many other. I use windows. 
if somebody of you know how to fix it I will be grateful.

**Comment by Mister-X-** [comment]
It's completely unrelated to aircrack-ng

**Reply by nqm322** [reply]
Yes, but what should i do? i already followed more than 5 guides on youtube how to fix it but i still getting the same error.
Changing DNS servers are not helping.
